changelog

```diff
@@ -1,3 +1,136 @@
+      * Adjusted the conversation in "FW Epilogue: Ijs and Katya" to better reflect the current state of the game. (@hexagonrecursion)
+      * Changed the description of how deep the ocean is in "Dangerous Games 2" to be more reasonable. (@ziproot)
+      * Expanded the conversation in "Sheragi Archaeology: The Box 7b" to include a "black box" joke. (@alexrovw)
+      * Diversified some assisting boarding missions. (@Daeridanii1)
+      * Changed how Sayari refers to humans when translating for the Wanderers in "Wanderers: Translation Machine". (@The-Legendary-M)
+    * Graphics:
+      * Updated Ka'het outfit graphics. (@beccabunny)
+      * Added a sprite for the Deep Mug. (@Anarchist2)
+      * New unique projectile and hit effect graphics for the Ionic Turret. (@RisingLeaf)
+      * Lowered the brightness of the center of the map sprite for better readability. (@RisingLeaf)
+    * Balance:
+      * Nerfed the Inferno and Blaze-pike turrets as they were a bit too strong. (@Quantumshark)
+        * Inferno
+          * Turn rate reduced from 2 to 1.3.
+          * Burn damage reduced from 0.9 to 0.75.
+        * Blaze-pike
+          * Turn rate reduced from 2.8 to 2.1.
+          * Burn damage reduced from 0.45 to 0.4.
+      * Nerfed the Flamethrower again. (@Quantumshark)
+        * Firing fuel increased from 0.25 to 0.4.
+        * Heat damage reduced from 150 to 125.
+      * Nerfed the Shield Refactor Module so that ships with larger outfit space can't abuse it. (@Arachi-Lover)
+        * Cost increased from 2843000 to 3248000.
+        * Decreased "hull repair rate" from -0.2 to -0.25.
+        * Replaced the "shield energy" attribute (at a value of 0.24) with the "shield energy multiplier" attribute (at a value of 0.2).
+    * Other:
+      * Removed the Quarg atrocity mission and added the proper atrocities to the Quarg governments. (@TomGoodIdea)
+      * Reworked and removed some overly-wordy and complex hails. (@bene-dictator)
+      * Locelace Labs will now sell Typhoon Torpedo outfits shortly after they are introduced. (@Quantumshark)
+      * Small wording tweaks in the intro mission set. (@Hurleveur)
+      * Various adjustments to intro Gegno content. (@Saugia)
+  * Game mechanics:
+    * New mechanics:
+      * Added gamerules for depreciation-related constants. (@1010todd)
+      * Added the ability to define custom spaceports with custom services. (@quyykk)
+      * Created two new attributes, "acceleration multiplier" and "turn multiplier", which allow the adjustment of a ship's acceleration or turn. (@Quantumshark)
+      * Engine hardpoints can now be given the "gimbal" attribute to rotate engine flares when thrusting and turning. (@Azure3141)
+      * The raid deterrence of a weapon now accounts for all combat damage types. (@Hurleveur)
+      * Added a new attribute, "use crew equivalent as crew", which enables the counting of "crew equivalent" as crew alongside a normal crew on a ship. (@Hurleveur)
```

Comment:
```suggestion
      * Added a new attribute, "use crew equivalent as crew", which causes the "crew equivalent" attribute to count as the total crew of the ship instead of adding to it for reputation purposes. (@Hurleveur)
```